rsmtool/notebooks/summary/model.ipynb

```diff
@@ -24,16 +24,20 @@
    "source": [
     "def summarize_models(model_list):\n",
     "    summs = []\n",
+    "    coefs = []\n",
     "    for (model_id, config, csvdir) in model_list:\n",
-    "        coef_file = os.path.join(csvdir, '{}_betas.csv'.format(model_id))\n",
-    "        if os.path.exists(coef_file):\n",
-    "            coefs = pd.read_csv(coef_file)\n",
-    "            model_summary = pd.DataFrame({'N features': [len(coefs)],\n",
-    "                                          'N negative': len(coefs.ix[coefs['standardized']<0]),\n",
+    "        coef_file = join(csvdir, '{}_betas.csv'.format(model_id))\n",
+    "        if exists(coef_file):\n",
+    "            df_coefs = pd.read_csv(coef_file)\n",
+    "            model_summary = pd.DataFrame({'N features': [len(df_coefs)],\n",
+    "                                          'N negative': len(df_coefs.ix[df_coefs['standardized']<0]),\n",
     "                                          'learner': config['model'],\n",
     "                                          'train_label': config['train_label_column']},\n",
     "                                         index=[model_id])\n",
     "            summs.append(model_summary)\n",
+    "            df_betas = pd.DataFrame({model_id : df_coefs['standardized'].values},\n",
+    "                                     index = df_coefs['feature'].values)\n",
+    "            coefs.append(df_betas)\n",
```

Comment:
No, coefs is a list defined in line 27 :) Could be renamed to `betas` for readability I suppose